web/frontend/src/components/base.vue

```diff
@@ -79,17 +78,20 @@ const setDirection = (dir: Directions) => {
   direction.value = dir;
 };
 
-const stop = () => {
+const stop = async () => {
+  const bc = new BaseClient(props.client, props.name, { requestLogger: rcLogConditionally });
```

Comment:
FYI @stevebriskin - we can can pass request loggers to the SDK component clients this way now

Reply:
nice! what kind of values does requestLogger take? 
`rcLogConditionally` is not a great name in this context any more.

Reply:
any `(unknown) => void`